contracts/custom/DecentVotesERC20.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+import { VotesERC20 } from "../VotesERC20.sol";
+
+contract DecentVotesERC20 is VotesERC20 {
+
+    error NotDelegatable();
+
+    function delegates(address account) public view override returns (address) {
+        return account;
```

Comment:
always just returns the account you're requesting as who it's delegated to

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/23b9ad8bd15638d2c335bd4ed1c74d825f61dc6a/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol#L80

Reply:
very clean solution here ser

Reply:
Nice. We also talked about the voting escrow token being non-transferrable. @adamgall  is that correct?